force-app/main/default/classes/AddNewOfferTest.cls

```diff
@@ -0,0 +1,103 @@
+/**
+ * @author Leszek Szczotka
+ * @date 03/01/2022
+ * @description Class created to test possibility of adding only one phisycal offer to one salon.
+ */
+@isTest
+public class AddNewOfferTest {
+  /**
+   * @author Leszek Szczotka
+   * @date 13.01.2022
+   * @description test method checking getting error when physical offer is already in one salon
```

Comment:
To wyglada jak test updatu na pojezdzie a nie insertu ofert a dodatkowo patrzac na ten test mozemy miec 200 takich samych ofert fizycznych to powino byc zablokowane nie tylko duplikaty z tym co jest w bazie danych ale rowniez z tym co wchodzi do bazy danych e.g. sprawdz czy mozesz zinsertowac liste ofert gdzie obydwie oferty wskazuja na ten sam pojazd ale na inne salony i sa fizyczne)